x-pack/legacy/plugins/maps/public/layers/fields/top_term_percentage_field.ts

```diff
@@ -22,6 +22,10 @@ export class TopTermPercentageField implements IESAggField {
     return this._topTermAggField.getSource();
   }
 
+  getAggType(): AGG_TYPE {
```

Comment:
Remove, `getAggType` should not be part of IESAggField interface. Its an implementation detail of ESAggField class